artdaq/DAQrate/FragmentBuffer.cc

```diff
@@ -375,6 +375,38 @@ void artdaq::FragmentBuffer::getDataBufferStats(Fragment::fragment_id_t id)
 	                          << ", sz=" << dataBuffer->DataBufferDepthBytes.load() << "/" << maxDataBufferDepthBytes_;
 }
 
+//-----------------------------------------------------------------------------
+// P.Murat: return stat reports as a string
+//-----------------------------------------------------------------------------
+void artdaq::FragmentBuffer::getStatReport(std::string& Report) {
```

Comment:
This should be `std::string artdaq::FragmentBuffer::getStatReport()`

Reply:
no problem